client/components/category/SingleCategory.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { selectCategory } from '../../redux/singleCategory'
+
+class SingleCategory extends Component {
+    componentDidMount() {
+        //this.props.selectCategory(this.props.selectedId)
+    }
+    render() {
+        const { selectedCategory } = this.props;
+        return (
+            <div id = "singleCategory">
+                <h1>Single Category</h1>
```

Comment:
Should this be category.title - is this rendering the name of the selected category? 

Also, is the purpose of this page to view a category and all of the costumes under that category? if that is the case, i think we may need to do a map in this and iterate over the costumes belonging to the category (which would be sent back via an "includes" i think from routes). does this make sense? please let me know if I am misunderstanding what we are doing with the page - very possible!!